frontend/sentiment-dashboard/src/app/auth/change-password/change-password.component.html

```diff
@@ -0,0 +1,13 @@
+<form [formGroup]="changePasswordForm" (ngSubmit)="onSubmitPassword(changePasswordForm.value)">
+  <div class="form-group">
+    <label for="oldPassword">Old password</label>
+    <input id="oldPassword" class="form-control col-6" type="password" formControlName="oldPassword">
+  </div>
+  <div class="form-group">
+    <label for="newPassword">New password</label>
+    <input id="newPassword" class="form-control col-6" type="password" autocomplete="new-password" formControlName="newPassword">
+  </div>
+  <p *ngIf="error" class="text-danger">{{error}}</p>
```

Comment:
Could we use a danger alert here? https://getbootstrap.com/docs/4.0/components/alerts/ That would make it more consistent with the rest of the UI.